src/app/shared/data-resource/data-resource-detail/data-resource-detail.component.html

```diff
@@ -1,17 +1,88 @@
+                      (switchToTable)="onSwitchToTable()"
+                      (dataResourceChanged)="dataResourceChanged.emit($event)">
+  </data-resource-data>
+  <div class="my-5">
+  </div>
+  <ng-container *ngIf="{commentsCount: commentsCount$ | async} as data">
+    <comments-panel *ngIf="resource && dataResource?.id"
+                    [maxCommentsCount]="1"
                     [permissions]="permissions"
-                    [constraintData]="constraintData$ | async"
-                    [workspace]="workspace$ | async"
-                    [ignoreSettingsOnReadPermission]="ignoreSettingsOnReadPermission"
-                    [query]="query"
-                    [toolbarRef]="toolbarRef"
-                    [preventEventBubble]="preventEventBubble"
-                    [attributeSettings]="attributeSettings"
-                    (attributeTypeClick)="onAttributeTypeClick($event)"
-                    (attributeFunctionCLick)="onAttributeFunctionClick($event)"
-                    (removeDocument)="onRemove()"
-                    (switchToTable)="onSwitchToTable()"
-                    (dataResourceChanged)="dataResourceChanged.emit($event)">
-</data-resource-data>
+                    [resourceType]="resourceType | attributeResourceTypeToResourceType"
+                    [resourceId]="dataResource.id"
+                    (newCommentClick)="editNewComment()">
+    </comments-panel>
+    <div *ngIf="data.commentsCount > 1">
+      <button class="btn btn-sm btn-success font-weight-bold my-3 mx-1"
+              i18n="@@document.detail.detail.comments.seeAll"
+              (click)="selectedTab = detailTabType.Comments">
+        See All {{data.commentsCount}} Comments &gt;
+      </button>
+    </div>
+  </ng-container>
+</ng-container>
+
+<ng-container *ngIf="selectedTab === detailTabType.Links && resourceType === collectionResourceType">
+  <div>
+    <!-- TODO Create Link Button -->
+    <links-accordeon class="d-block"
+                *ngIf="resource && dataResource?.id"
+                [document]="dataResource"
+                [preventEventBubble]="preventEventBubble"
+                [ignoreSettingsOnReadPermission]="ignoreSettingsOnReadPermission"
+                [allowSelectDocument]="allowSelectDocument"
+                [collection]="resource"
+                [viewSettings]="viewSettings"
+                (documentSelect)="documentSelect.emit($event)">
+    </links-accordeon>
+  </div>
+</ng-container>
+
+<ng-container *ngIf="selectedTab === detailTabType.Comments">
+  <comments-panel *ngIf="resource && dataResource?.id"
+                  [permissions]="permissions"
+                  [resourceType]="resourceType | attributeResourceTypeToResourceType"
+                  [resourceId]="dataResource.id"
+                  [startEditing]="startEditing"
+                  (onSaveComment)="startEditing = false"
+                  (onCancelComment)="startEditing = false">
```

Comment:
Same as above. This is not very good pattern to work with change detection correctly. I'm always using `BehaviorSubject` in these situations. Maybe it can work without it, but this is extremely dangerous.